schemas/client.yml

```diff
@@ -49,6 +49,7 @@
     - name: mac
       level: core
       type: keyword
+      example: 00-00-5E-00-53-23
```

Comment:
Thanks for using the range reserved for documentation purposes here.

Reply:
Thanks. A bit of history why I use them. I wanted to avoid "biased" addresses, ref https://github.com/ansible/ansible/issues/17479 (also contains a summary what addresses and names reserved for docs exist).